### DISCUSSION TITLE
Floating Preview.

### DISCUSSION THREAD

**Discussion by semi710** [nvim-tree/nvim-tree.lua#2700] [Q&A] [answered]
How can i have preview like this as in https://github.com/nvim-tree/nvim-tree.lua/issues/135#issuecomment-1586356684
as in this issue, the preview is in floating mode.
Is there any way to achieve this? And map `K` to show the preview and again pressing the `K` should put cursor in preview to scroll and `q` to exit from preview?

**Comment by alex-courtis** [comment, marked as answer]
That is a full [Extension Plugin nvim-tree.lua-float-preview](https://github.com/nvim-tree/nvim-tree.lua/wiki/Extension-Plugins#nvim-treelua-float-preview) - please install it and configure mappings as directed.
